src/applications/personalization/dashboard/components/benefit-application-drafts/MissingApplicationHelp.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const MissingApplicationHelp = () => (
+  <va-accordion uswds open-single="true" data-testid="missing-application-help">
```

Comment:
FYI specifying `uswds` is no longer required.

Reply:
nice, thanks! I'll remove these in my next PR. I have 1 or 2 more "Form Status" tickets I'm working on next 👍